src/main/java/com/twitter/clientlib/ApiClient.java

```diff
@@ -225,7 +225,7 @@ private void init() {
         json = new JSON();
 
         // Set default User-Agent.
-        setUserAgent("twitter-api-java-sdk/2.0.3");
+        setUserAgent("twitter-api-java-sdk/4.0.1");
```

Comment:
I would be wary of changing this as it may be that we need this user agent to be whitelisted by customers in some fashion. We can internally version it what we wnat I would say and keep the user agent as it was. I notice this was how Joanna did it in the past.